test/cql-pytest/cassandra_tests/functions/cast_fcts_test.py

```diff
@@ -15,8 +15,6 @@ def testInvalidQueries(cql, test_keyspace):
     with create_table(cql, test_keyspace, "(a int primary key, b text, c double)") as table:
         assertInvalidMessage(cql, table, "cannot be cast to", "SELECT CAST(a AS boolean) FROM %s")
 
-# Marked cassandra_bug because of CASSANDRA-18647 (see comment below)
```

Comment:
I looked again, and you're right - this test will actually pass on recent versions Cassandra because this bug was fixed after I reported it. So you're indeed right to remove this line now. Thanks!